.github/workflows/ruff-linter.yml

```diff
@@ -1,19 +0,0 @@
----
```

Comment:
Why is Ruff workflow removed? I see it's part of MegaLinter now, but those run different versions and it's not using the Ruff config for this project.